### PR TITLE
LPS-188190 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-188190](https://liferay.atlassian.net/browse/LPS-188190)**

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path [hunk 1]
```diff
@@ -605,8 +605,8 @@
 	<td></td>
 </tr>
 <tr>
-	<td>ICON_TRASH_ROLE_TYPE</td>
-	<td>//div[@class='panel-body']//input[contains(@id,'notificationName') and contains(@value,'')]/../..//*[contains(@class, 'icon-trash')]</td>
```
Comment (davisaints) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path:609]: This doesn't work if it has more than one trash can icon. 

As a solution, I implement a index on the path.

**Comment by patriciajeanperez** [comment]
Hey @davisaints, can you attach the passing test to the ticket?

**Comment by davisaints** [comment]
Ohh sorry! It's done now @patriciajeanperez

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:forward

**Pull request closed without merging**